### ISSUE TITLE
Can I use a corpus that have a new sentence instead of word every line?

### ISSUE THREAD

**Issue by boy977** [Belval/TextRecognitionDataGenerator#102]

**Comment by Belval** [comment]
To be clear, you have multiple custom fonts that you want to use when generating?

**Comment by boy977** [comment]
Yes. Particularly I'm working with Japanese text and I have a problem with other tools that sometimes the text is not rendered right because of the font does not support all the characters. Do you have a solution for that?

**Comment by Cabbagehust2507** [comment]
@boy977 maybe your fonts not be support for all words. I met the same problem but when I change my fonts, it was worked for me. :)

**Comment by boy977** [comment]
@Cabbagehust2507 But I want to use all the fonts to render. They are 10 fonts that support Japanese in linux but some of them do not support all the characters. So if the tool can provide a way to check if a character is rendered wrong and remove it

**Comment by Belval** [comment]
That's the part I didn't understand from your initial issue.

I can confirm that there is no support for that right now, but it's a feature that I wanted to include for a long time.

If you want to work on it, I would be willing to merge a PR for it. If not, I can have a look in the coming week.

**Comment by boy977** [comment]
No promise but I will try to work on it
